xapian-core/api/omenquire.cc

```diff
@@ -452,6 +464,23 @@ MSet::Internal::read_docs() const
     requested_docs.clear();
 }
 
+void
+MSet::Internal::set_new_weights(double wt, int i) {
+    item_set_weight(items[i], wt);
+}
+
+void
+MSet::Internal::item_set_weight(Xapian::Internal::MSetItem& item_,double wt_)
+{
+    item_.wt = wt_;
+}
+
+void
+MSet::Internal::re_rank()
```

Comment:
Having consistency across the API is better, so `sort_by_relevance()` would be the right choice here since we already have a similar-named function working in the same way.